README.md

```diff
@@ -1,79 +1,66 @@
 Takeaway Challenge
 ==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
 
- ```
+Here is my work so far on this and the below user stories go someway to explaining what this code is supposed to do (these tests are also in the spec/feature_tests.rb file).
 
-Instructions
--------
+Gems used
+---------
+Twilio API
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
 
-Task
------
+Additional Notes
```

Comment:
This is super helpful as it helps me to see that you're thinking about code quality. There will be time constraints on any project and I think you've taken a sensible approach here; starting out with one class and (planning to) abstract some behaviour into a new class / classes.  Deciding when exactly to make this abstraction will be influenced by a few things and getting it right will come with experience.